new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -42,6 +45,40 @@
 // directly call the methods in the impl classes
 public class TestActions {
 
+  // ----------------------   pod  ---------------------------------
+  
+  /**
+   * Get the creationTimestamp for a given pod with following parameters.
+   * @param namespace in which to check for the pod existence
+   * @param labelSelector in the format "weblogic.domainUID in (%s)"
+   * @param podName name of the pod 
+   * @return creationTimestamp from metadata section of the Pod
+   */
+  public static String getPodCreationTime(String namespace, String labelSelector, String podName) {
+    return Pod.getPodCreationTime(namespace,labelSelector,podName);
+  }
+ 
+  /**
+   * Get the Pod object with following parameters.
+   * @param namespace in which to check for the pod existence
+   * @param labelSelector in the format "weblogic.domainUID in (%s)"
```

Comment:
labelSelector in the format "weblogic.domainUID in (%s)" , can be null

Reply:
yes, it can be a null string like  ""